packages/core/test/unit/html/includePanelProcessor.test.ts

```diff
@@ -87,6 +87,57 @@ test('includeFile replaces <include src="doesNotExist.md" optional> with empty <
   expect(result).toEqual(expected);
 });
 
+test('includeFile import footnote from hash', async () => {
+  const indexPath = path.resolve('index.md');
+
+  const index = [
+    '<include src="include.md#exists"/>',
+    '',
+  ].join('\n');
+
+  const include = [
+    '<div id="exists">',
+    '',
+    'text^[footnote]',
+    '</div>',
+    '',
+  ].join('\n');
+
+  const json = {
+    'index.md': index,
+    'include.md': include,
+  };
+
+  fs.vol.fromJSON(json, '');
+
+  const nodeProcessor = getNewDefaultNodeProcessor();
+  const result = await nodeProcessor.process(indexPath, index);
+  // to make lint happy
```

Comment:
Also could you write a better comment here? Why is lint complaining and what is done to fix it

Reply:
Fix in latest commit; no longer has this comment by writing in the style suggested by @kaixin-hc